src/plugins/dashboard/public/plugin.tsx

```diff
@@ -135,6 +147,15 @@ export class DashboardEmbeddableContainerPublicPlugin
     );
     uiActions.registerAction(changeViewAction);
     uiActions.attachAction(CONTEXT_MENU_TRIGGER, changeViewAction);
+    return {
+      getSavedDashboardLoader: () =>
```

Comment:
why do we need to expose `getSavedDashboardLoader` on the contract ? (why do we need multiple instances of it ? )

i think we should rather expose `savedDashboardLoader` directly 
```
{
   savedDashboardLoader: createSavedDashboardLoader({ ....})
}
```

Reply:
Made it a singleton, kept it as a fn though.